fecgov/openFEC
# Add committee_name into 3 committee reports MV and export files

## Summary (required)
House and Senate committee reports, presidential committee reports and PAC and party committee reports downloads do not include names, just IDs
so we need add committee_name column into csv download file.

- Resolves #[[3844]](https://github.com/fecgov/openFEC/issues/3844)

_Include a summary of proposed changes._
modify/create three report committee MV to add `committee_name` column:
a)ofec_reports_presidential_mv
b)ofec_reports_house_senate_mv
c)ofec_reports_pac_party_mv

## How to test the changes locally
1)api test locally
checkout branch, point to dev. 
modify models/reports.py:
ofec_reports_presidential_mv ==> ofec_reports_presidential_mv_jl
ofec_reports_house_senate_mv ==>ofec_reports_house_senate_mv_jl
ofec_reports_pac_party_mv ==>ofec_reports_pac_party_mv_jl

go to swagger under tag: **financial**
path: /reports/{committee_type}/

2)test download csv file
option 1, deploy your feature branch to dev space.
go to https://fec-dev-proxy.app.cloud.gov/data/browse-data/?tab=filings
click each of three committee type reports to export csv file to see `committee_name` column.

option 2, test locally:
1. get `cf env api` from dev space
export access_key_id="xxxxxxx"
export secret_access_key="xxxxxxx"
export bucket="xxxxxxx"
export region="xxxxxxx"

2. setup 4 terminals on local:
1)first terminal: Redis
`redis-server`
2)second terminal: Celery Worker
active virtual env
setup 4 env variables (see above)
`celery worker --app webservices.tasks --loglevel INFO `
3)Third terminal: openFEC (point dev DB)
active virtual env
setup 4 env variables (see above)
export SQLA_CONN=dev database
./manage.py runserver
4)Forth terminal: fec-cms (point your local api)
active virtual en
export DATABASE_URL=postgresql://:@/cfdm_cms_test
export FEC_API_URL=http://localhost:5000
export FEC_WEB_API_KEY=xxxxxx
export FEC_WEB_API_KEY_PUBLIC=xxxxxxx
export FEC_CMS_ENVIRONMENT=LOCAL

3. test export csv file for three committee type reports.
http://127.0.0.1:8000/data/browse-data/?tab=filings
click each of three committee type reports to export csv file to see `committee_name` column.

4. compare with production website export.

## Impacted areas of the application
List general components of the application that this PR will affect:
Filings and reports page:
https://www.fec.gov/data/browse-data/?tab=filings
